securesystemslib/signer/_sigstore_signer.py

```diff
@@ -268,3 +268,26 @@ def sign(self, payload: bytes) -> Signature:
             bundle_json["messageSignature"]["signature"],
             {"bundle": bundle_json},
         )
+
+    @classmethod
+    def import_github_actions(
+        cls, project: str, workflow_path: str, ref: Optional[str] = "refs/heads/main"
+    ) -> Tuple[str, SigstoreKey]:
+        """Convenience method to build identity and issuer string for import_() from
+        GitHub project and workflow path.
+
+        Args:
+            project: GitHub project
+            worfklow_path: GitHub workflow path
```

Comment:
typo fix plus added some examples to make using easier
```suggestion
            project: GitHub project name (example: "secure-systems-lab/securesystemslib") 
            workflow_path: GitHub workflow path (example: ".github/workflows/online-sign.yml")
```